src/compiler/crystal/command/format.cr

```diff
@@ -121,7 +121,7 @@ class Crystal::Command
         end
       elsif Dir.exists?(filename)
         filename = filename.chomp('/')
-        filenames = Dir["#{filename}/**/*.cr"]
+        filenames = Dir[::Path.new(filename, "**/*.cr").to_posix]
```

Comment:
**thought:** I'm wondering if it's a good idea to put wildchards into `Path`. It works, and it's probably correct to expect that it works. However, it might be semantically preferable to use plain string operations for stitching together a glob pattern.
All other locations in this PR use string interpolation as well.

```suggestion
        filename = ::Path.new(filename).to_posix
        filenames = Dir["#{filename}/**/*.cr"]
```

Reply:
`filename` may contain backslashes on Windows (`Crystal.normalize_path` doesn't replace them with forward slashes), so the `to_posix` is necessary

Reply:
I'm suggesting to apply `to_posix` *only* to `filename`, not the wildcard suffix which we know does not contain any backslashes.